File: Semestres/11.Febrero-Junio 2022/EstructurasDatosUSB/corte2/Tema2Arboles/MainArbol4.java
public class MainArbol4 {
    public static void main(String[] args){
        Arbol4 objArb = new Arbol4(
    1,
    new Arbol4(
        2,
        new Arbol4(6,null,null, null, null),
        new Arbol4(7,null,null,null,null),
        new Arbol4(8,null,null,null,null),
        null
    ),
    new Arbol4(3,new Arbol4(9,null,null,null,null),null,null,null),
    new Arbol4(4,new Arbol4(10,null,null,null,null),null,null,null),
    new Arbol4(5,new Arbol4(11,null,null,null,null),new Arbol4(12,null,null,null,null),null,null)
    );

    System.out.println(objArb.getLeft().getRight2().getKey());
    }
}

File: Semestres/11.Febrero-Junio 2022/EstructurasDatosUSB/corte2/Tema2Arboles/MainArbol4.py
from Arbol4 import Arbol4


objArb = Arbol4(
    1,
    Arbol4(
        2,
        Arbol4(6,None,None, None, None),
        Arbol4(7,None,None,None,None),
        Arbol4(8,None,None,None,None),
        None,
    ),
    Arbol4(3,Arbol4(9,None,None,None,None),None,None,None),
    Arbol4(4,Arbol4(10,None,None,None,None),None,None,None),
    Arbol4(5,Arbol4(11,None,None,None,None),Arbol4(12,None,None,None,None),None,None)
)

print(objArb.left.right2.key)